tests/php/src/Editor/EditorSupportTest.php

```diff
@@ -165,4 +143,32 @@ public function test_maybe_show_notice_with_cpt_supporting_gutenberg_but_not_amp
 		unset( $GLOBALS['current_screen'] );
 		unset( $GLOBALS['wp_scripts'] );
 	}
+
+	/** @covers ::maybe_show_notice() */
+	public function test_maybe_show_notice_for_gutenberg_4_9() {
+		if ( ! defined( 'GUTENBERG_VERSION' ) || version_compare( GUTENBERG_VERSION, '4.9.0', '>' ) ) {
+			$this->markTestSkipped( 'Test only applicable to Gutenberg v4.9.0 and older.' );
+		}
+
+		register_post_type(
+			'my-gb-post-type',
+			[
+				'public'       => true,
+				'show_in_rest' => true,
+				'supports'     => [ 'editor' ],
+			]
+		);
+
+		set_current_screen( 'edit.php' );
+		$post = $this->factory()->post->create( [ 'post_type' => 'my-gutenberg-post-type' ] );
```

Comment:
This test is failing, at least because `my-gb-post-type` is the post type defined but `my-gutenberg-post-type` is the post type created, but neither of these are among the `AMP_Post_Type_Support::get_eligible_post_types()`.